ui/app/components/auth/page.hbs

```diff
@@ -29,21 +27,9 @@
             <Icon @name="vault" @size="24" @stretched={{true}} />
           </div>
         </div>
-        <div class="is-flex-row">
-          {{#if this.mfaAuthData}}
-            <Hds::Button
-              @text="Back to login"
```

Comment:
Button lives in the `mfa-form.hbs` component now, design signed off on this change ✔️